app/assets/javascripts/admin.js

```diff
@@ -39,4 +38,7 @@ $(document).on('ready', function(e) {
     $feedback.find('.message').text(message);
     $feedback.show();
   });
+
+  $('select').not('.disabled').material_select();
+  Materialize.updateTextFields();
```

Comment:
What do these function do? Do they simply apply Materialize styling to these form fields.